src/runtime/capi.cpp

```diff
@@ -667,6 +667,8 @@ void checkAndThrowCAPIException() {
         PyErr_Clear();
 
         // This is similar to PyErr_NormalizeException:
+        // TODO: why are we doing something "similar" to PyErr_NormalizeException instead of calling
+        // PyErr_NormalizeException?
```

Comment:
I think the "why" is just because we started off doing something that wasn't quite the same as CPython and we haven't taken the time to make it exactly the same :)  I think there's a good todo in here for making them similar (I think there are still some issues around this)